extensions/cache/runtime/src/main/java/io/quarkus/cache/runtime/CacheInterceptionContext.java

```diff
@@ -0,0 +1,23 @@
+package io.quarkus.cache.runtime;
+
+import java.lang.annotation.Annotation;
+import java.util.ArrayList;
+import java.util.List;
+
+public class CacheInterceptionContext<T extends Annotation> {
```

Comment:
It would be nice if this class was immutable... WDYT?

Reply:
I agree, thanks for the suggestion. It took care of it in #14111.